webrecorder/browsertrix
# refactor: Migrate LiteElements

Related to https://github.com/webrecorder/browsertrix/issues/1984

## Changes

- Migrates the following `LiteElement`s to `BtrixElement`:
  - `App` (`<browsetrix-home>`)
  - `Admin` (`<btrix-home`)
  - `Dashboard` (`<btrix-dashboard>`)
  - `LogInPage` (`<btrix-log-in>`)
- Replaces custom light DOM `btrix-input` with `sl-input` since password managers should work now.

## Manual testing

1. Log out and go to log in page
2. Click email and password input. Verify that your password manager/autofill works
3. Log in as regular user. Verify org dashboard renders as expected
4. Log out and log as suepradmin. Verify superadmin dashboard renders as expected

## Context

This will allow usage of lit reactive controllers in more components, for example to react to lit locale changes.
